packages/storage-plus/src/indexes/multi.rs

```diff
@@ -24,14 +25,16 @@ use crate::{Bound, Index, Prefix, Prefixer, PrimaryKey};
 /// The MultiIndex definition must include a field for the pk. That is, the MultiIndex K value
 /// is always a n-tuple (n >= 2) and its last element must be the pk.
 /// The index function must therefore put the pk as last element, when generating the index.
-pub struct MultiIndex<'a, K, T> {
+/// The optional PK type defines the type of Primary Key deserialization.
+pub struct MultiIndex<'a, K, T, PK = ()> {
     index: fn(&T, Vec<u8>) -> K,
     idx_namespace: &'a [u8],
     idx_map: Map<'a, K, u32>,
     pk_namespace: &'a [u8],
+    _phantom: PhantomData<PK>,
```

Comment:
It needs to be initialized in constructor anyway, and `PhantomData` shows that it's Phantom anway - I'd argue it's "rusty" to keep name without unused mark `_`:
```suggestion
    phantom: PhantomData<PK>,
```